update-formula.sh

```diff
@@ -16,28 +17,35 @@ cp $FORMULA pachctl@$MAJOR_MINOR.rb
 FORMULA="pachctl@$MAJOR_MINOR.rb"
 
 # Linux doesn't allow a space after the -i flag in sed. OSX needs it.
-
+AMDURL=$GITHUB_HOSTING_ROOT"/v"$VERSION"/pachctl_"$VERSION$MACINTEL_BINARY_SUFFIX
+ARMURL=$GITHUB_HOSTING_ROOT"/v"$VERSION"/pachctl_"$VERSION$MACARM_BINARY_SUFFIX
 # Put the file in place
 ### replace binary URL
 if [[ "$OSTYPE" == "darwin"* ]]; then
-  sed -i '' 's@url ".*"@url "'"$GITHUB_HOSTING_ROOT"'/v'"$VERSION"'/pachctl_'"$VERSION$MAC_BINARY_SUFFIX"'"''@' $FORMULA
+  sed -i '' -e "s|AMDURLPH|${AMDURL}|g" $FORMULA
+  sed -i '' -e "s|ARMURLPH|${ARMURL}|g" $FORMULA
 else
```

Comment:
I thought homebrew was only for mac?

Reply:
Looks like it supports linux and windows linux too.

Reply:
no this is for when this script gets run- local on mac vs on linux in circle